spec/reactors/consensus.md

```diff
@@ -0,0 +1,261 @@
+# Consensus Reactor
+
+Consensus reactor handles message propagation for 4 different channels, namely, `StateChannel`, `DataChannel`, `VoteChannel`, and `VoteSetBitsChannel`.
```

Comment:
[Note to the Reviewers] This document contains additional information and question presented as comments. If you possess insights regarding the questions, kindly incorporate them into your review. Regarding the other comments containing side information, I am currently examining them. Once confirmed, I intend to integrate them into the main text. However, if you have any relevant input on these matters, please share your thoughts in the comments. Thansk!